src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,3 +1,37 @@
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import { Posts } from '../../types/postsAll';
+
+type Props = {
+  post: Posts;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const commentsLists = post.comment;
+  const commentsListLength = commentsLists.length;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        {' Posted by  '}
```

Comment:
```suggestion
        Posted by
```
you can use a plain text in your jsx